.travis.yml

```diff
@@ -43,6 +44,7 @@ install:
 - travis_retry composer install $COMPOSER_ARGS
 
 script:
+- if [[ $EXECUTE_TEST_PHPSTAN == 'true' ]]; then ./vendor/bin/phpstan analyse -l 7 src ; fi
```

Comment:
Should we run in on the test folder too? We may need phpstan-phpunit for that.

Reply:
I agree. And it's better to call phpstan on current directory. Filters can be set on phpstan configuration if needed.